static/css/style.css

```diff
@@ -0,0 +1,27 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+body {
+    background-image: url(../images/background.jpg);
+    background-position-x: center;
+    background-position-y: center;
+    background-size: 65% 70%;
+    background-repeat: no-repeat;
+    height: 100vh;
+    width: 100vw;
+    font-family: 'Poppins', sans-serif;
+    
+}
+
+.container-fluid {
+  text-align: center;
+  margin: 0 auto;
+  display: block;
+  position: absolute;
+  top: 55%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.btn {
+  margin-top: 0.8rem;
+}
```

Comment:
good concise css - could you move the background image from the body to the container, maybe help with responsiveness?